Boltdir/site-modules/releng_tasks/files/http_request.rb

```diff
@@ -0,0 +1,58 @@
+require 'net/http'
+require 'uri'
+require 'json'
+require 'openssl'
+
+# A fit-for-most-purposes, MRI-compatible HTTP/S swiss army knife method
```

Comment:
MRI = Ruby MRI?

Reply:
@lnemsick-simp Yeah, that's standard shorthand.